sdk/hybridaks/Azure.ResourceManager.HybridContainerService/src/Generated/AgentPoolResource.cs

```diff
@@ -0,0 +1,504 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+
+namespace Azure.ResourceManager.HybridContainerService
+{
+    /// <summary>
+    /// A Class representing an AgentPool along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct an <see cref="AgentPoolResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetAgentPoolResource method.
+    /// Otherwise you can get one from its parent resource <see cref="ProvisionedClustersResponseResource" /> using the GetAgentPool method.
+    /// </summary>
+    public partial class AgentPoolResource : ArmResource
```

Comment:
`AgentPool` is a common resource name. Please add a prefix to make it more descriptive. Either use the `prepend-rp-prefix` if `HybridContainerServiceAgentPool` is a good name to use or use `rename-mapping` to use another name in  `autorest.md`. You can check rename configuration examples in [ContainerService autorest.md](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/containerservice/Azure.ResourceManager.ContainerService/src/autorest.md)

Reply:
Thanks for the suggestion, I've renamed it now